cp3pt0-deployment/setup_tenant.sh

```diff
@@ -78,6 +79,10 @@ function parse_arguments() {
         --enable-licensing)
             ENABLE_LICENSING=1
             ;;
+        --with-minimal-rbac)
```

Comment:
Considering we have two cases, would you mind adding one more var `MINIMAL_RBAC_ENABLED` and assigned with 1 when the customers do require minimal rbac?
```
--with-minimal-rbac)
    shift
    MINIMAL_RBAC_ENABLED=1
    MINIMAL_RBAC=$1
    ;;
```

Reply:
done